src/utils/shared_functions.py

```diff
@@ -3,6 +3,7 @@
 import glob
 import inspect
 import os
+import traceback
```

Comment:
@GregoryPetrochenkov-NOAA is the `traceback` module needed here?

Reply:
I went ahead and removed this import